src/borg/archiver.py

```diff
@@ -76,10 +76,14 @@ def wrapper(self, args, **kwargs):
             if argument(args, fake):
                 return method(self, args, repository=None, **kwargs)
             elif location.proto == 'ssh':
+                # RemoteRepository has no notion of append_only
+                # It can't be implemented without cooperation with the server
+                # We don't want to break the RPC API though
```

Comment:
Oh. I missed that. In that case, mind if I mess with the negotiation to add a downgradable protocol version?